tests/codegen/ccode/scripts/ifs.py

```diff
@@ -0,0 +1,93 @@
+
+x = 0
+y = 0
+z = 3
+
+if 1<0:
+    x=4
+
+if (x==2 or y==1) and z>3 :
+    x = 3
+    y = 4
+
+while x>2 and y<1:
+    x = x-1
+    y = 4
```

Comment:
Shouldn't this belong to file `whiles.py`?

Reply:
I agree but I suspect that these crept in as they are present in `tests/codegen/fcode/scripts/ifs.py`